frontend/lib/laletterbuilder/letter-builder/choose-letters.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import { t } from "@lingui/macro";
+
+import { toDjangoChoices } from "../../common-data";
+import { RadiosFormField } from "../../forms/form-fields";
+import { li18n } from "../../i18n-lingui";
+import { ProgressButtons } from "../../ui/buttons";
+import {
+  LetterChoices,
+  getLetterChoiceLabels,
+} from "../../../../common-data/la-letter-builder-letter-choices";
+import { SessionUpdatingFormSubmitter } from "../../forms/session-updating-form-submitter";
+import Page from "../../ui/page";
+import { MiddleProgressStep } from "../../progress/progress-step-route";
+import { LaLetterBuilderChooseLetterMutation } from "../../queries/LaLetterBuilderChooseLetterMutation";
+
+export const LaLetterBuilderChooseLetterStep = MiddleProgressStep((props) => {
+  return (
+    <Page title={li18n._(t`Letter type you'd like to send`)}>
+      <SessionUpdatingFormSubmitter
```

Comment:
If this page is simply navigating a user to the right path (depending on letter type), I'm wondering if it even needs to be a Session-updating form at all? Couldn't it be just a list of links? Or, if you wanted to keep the radio buttons, you could just save the radio button selection on the front end, and that dynamically determines what the "next" button goes to?